src/components/Box/Box.css

```diff
@@ -0,0 +1,8 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+@import "src/css/variables/colors.css";
+
+.y-box {
+  position: relative;
+  border-radius: 2px;
+  border: 1px solid $color__indieRock;
```

Comment:
@swese44 should we push for semantic colors everywhere?

Reply:
yes, please use $borderColor here by importing colors.css instead of _yammer_palette.css